stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/state/messages/DeletedMessagesVisibility.kt

```diff
@@ -0,0 +1,21 @@
+package io.getstream.chat.android.compose.state.messages
+
+/**
+ * Represents the behavior of deleted messages in the list, if they should show for some users or no one.
+ */
+public enum class DeletedMessagesVisibility {
```

Comment:
Could we align this with UI Comp & iOS? 

```
public enum class DeletedMessageVisibility {

    ALWAYS_HIDDEN,
    VISIBLE_FOR_CURRENT_USER,
    ALWAYS_VISIBLE
}
```

Reply:
@MarinTolic https://github.com/GetStream/stream-chat-android/pull/3298#discussion_r843663261